docs/modules/network-partitioning/pages/split-brain-recovery.adoc

```diff
@@ -5,7 +5,8 @@ When a split is detected, the side that will initiate the merge process is decid
 This decision is based on the cluster size; the smaller cluster, by
```

Comment:
Maybe this paragraph could read something like this to get over the idea of smaller and larger clusters:

This decision is based on the cluster size. The cluster with the smallest number of members is merged into the larger cluster. If the clusters…

Reply:
Let's not mention about smaller/larger cluster concept, since as Josef suggested that the split could be 1:1 or 2:2. I'd go with the "healing cluster" as he suggested.